programs/locked-voter/src/instructions/open_dispute.rs

```diff
@@ -0,0 +1,84 @@
+use crate::*;
+/// Accounts for [voter::create_move_request].
+#[derive(Accounts)]
+pub struct OpenDispute<'info> {
+    /// [Escrow].
+    #[account(
+        mut,
+        has_one = owner,
+        has_one = locker
+    )]
+    pub escrow: Box<Account<'info, Escrow>>,
+    pub locker: Box<Account<'info, Locker>>,
+    /// [Recovery_Token_Account]
+    pub recovery_key: Signer<'info>,
+
+    /// [Request].
+    #[account(
+        init_if_needed,
+        seeds = [
+            b"Request".as_ref(),
+            escrow.key().as_ref(),
+            recovery_key.key().as_ref(),
+        ],
+        bump,
+        payer = feepayer,
+        space = MoveRequest::LEN
+    )]
+    pub request: Box<Account<'info, MoveRequest>>,
+
+    /// Authority of the [Old_Escrow].
+    pub owner: Signer<'info>,
+
+    /// Fee payer, and also new owner
+    #[account(mut)]
+    pub feepayer: Signer<'info>,
```

Comment:
1. Why in this context, we need both recovery_key and feepayer? 
request is PDA from recovery_key.key(), but actually it stores request. recovery_key = feepayer? 

2. If feepayer is new owner, then just use the field `new_owner`, assertion that new_owner != owner